src/Tests/SmsFrameworkSettingsTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\sms\Tests\SmsFrameworkSettingsTest.
+ */
+
+namespace Drupal\sms\Tests;
+
+use Drupal\Core\Url;
+
+/**
+ * Tests SMS Framework settings form.
+ *
+ * @group SMS Framework
+ */
+class SmsFrameworkSettingsTest extends SmsFrameworkWebTestBase {
+
+  /**
+   * Test changing verification path.
+   */
+  public function testSettingsForm() {
+    $account = $this->drupalCreateUser([
+      'administer smsframework',
+    ]);
+    $this->drupalLogin($account);
+
+    // Test invalid path.
+    $edit = [
+      'default_gateway' => 'log',
+      'pages[verify]' => $this->randomMachineName() . '/' . $this->randomMachineName(),
+    ];
+    $this->drupalPostForm(Url::fromRoute('sms.settings'), $edit, 'Save configuration');
+    $this->assertRaw(t("Path must begin with a '/' character."));
```

Comment:
I forget the whole reason why, but its something to do with the parent site leaking language. Using t in asserts is acceptable, even if not using placeholders. But if you render with t, you can (should?) use it in tests.

Search "assertRaw(t(" or assertText(t(" for plenty of instances.